.github/workflows/release.yml

```diff
@@ -16,15 +16,31 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       tag: ${{ steps.get-tag.outputs.tag }}
+      last: ${{ steps.get-last-tag.outputs.last }}
+      notes: ${{ steps.get-release-notes.outputs.notes }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-
+        
       - name: Get release tag
         id: get-tag
         run: |
           MERGES=$(gh pr list -B main -s merged --search "closed:>2021-02-28" -L 9999 | wc -l | awk '{print $1}')
           echo "::set-output name=tag::release-$MERGES"
+          
+      - name: Get last release
+        id: get-last-tag
+        run: |
+          LAST=$(gh release list -L 1 | awk '{print $1}')
+          echo "::set-output name=last::$LAST"
+          
+      - name: Get release notes
+        id: get-release-notes
+        run: |
+          NOTES=$(git log ${{ steps.get-last-tag.outputs.last }}..${{ steps.get-tag.outputs.tag }} --pretty="@%an - %b (%s)[%h]" --merges | sed -e 's!Merge pull request !!')
```

Comment:
I think we want `..HEAD`. We haven't created the tag at this point (that happens in the next step) so `git` wont know what this reference is.